src/stories/AddButton.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import AddButton from "../components/AddButton";
+
+export default {
+  title: "AddButton"
+};
+
+export const SubmitNewRestaurant = () => <AddButton></AddButton>;
```

Comment:
The story has no connection to restaurants. That's why I wouldn't name the story `SubmitNewRestaurant`. Instead, Default would be a good name